packages/babel-cli/src/babel/file.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import convertSourceMap from "convert-source-map";
-import defaults from "lodash/defaults";
+import defaults from "lodash.defaults";
```

Comment:
We should be able to remove `defaults`, yeah?

```
const res = await util.transform(
      cliOptions.filename,
      code,
      {
        ...babelOptions,
        sourceFileName: "stdin",
      },
);
```

Reply:
Possibly! - we'd want to be careful about the argument ordering to the spread operator:

```
> var defaults = {'option': 'valueFromDefault'};
> var cliArgs = {'option': 'commandLineArgument'};
> {...defaults, ...cliArgs}
{ option: 'commandLineArgument' }
> {...cliArgs, ...defaults}
{ option: 'valueFromDefault' }
```

Also a subtle difference is that `lodash.defaults` mutates the first argument and returns it as a value, whereas the spread operator returns a new shallow clone; that's probably fine and safe in most cases, but worth being careful about.

Reply:
In our case the mutation isn't a problem, since the first object is always defined inline in the call.

Reply:
Okie doke, thanks - https://github.com/babel/babel/pull/11797 contains an attempt to clean up the use of `lodash.defaults` from the codebase (including this callsite).